incubator-tools/docai_processor_visual_assessment_tool/docai_processor_visual_assessment.ipynb

```diff
@@ -0,0 +1,748 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #255.            page_number += 1
</pre><p>A lot of the code in this cell seems to be duplicated. It could be simplified down to make it easier to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Refactored the functions.